examples/simple_example.cc

```diff
@@ -83,5 +83,15 @@ int main() {
 
   delete db;
 
+  {
```

Comment:
I understand the intent for testing, but I don't think this fits the purpose of simple_example.cc

Reply:
Will remove. How about adding a `examples/write_batch_example.cc`?